core/src/main/groovy/noe/jbcs/utils/HttpdHelper.groovy

```diff
@@ -161,6 +161,12 @@ class HttpdHelper {
     } else {
       throw new RuntimeException("[${httpd.serverId}] Postinstall went wrong => ${postInstallOutput}")
     }
+
+    // Overrride the user that installed httpd
+    String OVERRIDE_INSTALL_AS_USER = Library.getUniversalProperty('OVERRIDE_INSTALL_AS_USER', '')
+    if(platform.isRHEL() && !OVERRIDE_INSTALL_AS_USER.isEmpty()) {
+      def result = commandExecution(["chown", "-R", "${OVERRIDE_INSTALL_AS_USER}:${OVERRIDE_INSTALL_AS_USER}", "${httpdBasedir}"], new File(httpdBasedir))
```

Comment:
Did you tested it? It is missing `sudo` rights so it probably fail.

Reply:
Yes, :smile: commandExecution(...) is a reference to the function Cmd.executeMethodBasedOnAdminPrivileges(...) that selects to use a `sudo` function not, depending on whether USE_ADMIN_PRIVILEGES is set true or false.